accord-core/src/main/java/accord/local/Node.java

```diff
@@ -687,6 +687,12 @@ public TxnId nextTxnId(Txn.Kind rw, Domain domain, Cardinality cardinality)
         return nextTxnId(rw, domain, cardinality, defaultMediumPath().bit());
     }
 
+    public TxnId nextTxnId(Txn.Kind rw, Domain domain, Cardinality cardinality, long minHlc)
+    {
+        Timestamp minTimestamp = Timestamp.fromValues(topology.epoch(), minHlc, 0, id());
```

Comment:
might be cleaner to invoke `newTxnId` directly with the epoch and uniqueNow(minHlc)?

Reply:
might also be more consistent to accept minHlc as first parameter to match Timestamp min variant below? Or could perhaps introduce a nextTxnId(minEpoch, minHlc, ... to accept both of these variants